scripts/vippy/rest.py

```diff
@@ -92,21 +99,26 @@ def make_table_formatter(all_types, data_type, prefix):
     is_enum = data_type.is_enum
     column_infos, cell_values = get_type_info(is_enum, prefix)
     keys, headers, widths = ([c[i] for c in column_infos] for i in range(3))
-    formatter = TableFormatter(all_types=all_types, headers=headers, keys=keys,
-                           widths=widths, cell_values=cell_values, prefix=prefix)
+    formatter = TableFormatter(
+        all_types=all_types,
+        headers=headers,
+        keys=keys,
+        widths=widths,
+        cell_values=cell_values,
+        prefix=prefix,
+    )
     return formatter
 
 
-
-def make_table(all_types, data_type, prefix):
+def make_standalone_table(all_types, data_type, prefix, mode):
```

Comment:
Previously there had been two functions named `make_table()`; this one and a method of a class.